include/swift/Basic/LangOptions.h

```diff
@@ -171,7 +171,7 @@ namespace swift {
     ///
 
     /// User-overridable language version to compile for.
-    version::Version EffectiveLanguageVersion = version::Version::getCurrentLanguageVersion();
+    version::Version EffectiveLanguageVersion = version::Version{5, 10};
```

Comment:
@shahmishal, do you plan to bump this to 6 soon? Otherwise, I'm seeing strange behavior like this when compiling code with the Swift 6 snapshots:
```
> cat check.swift
#if swift(>=6.0)
print("got to 6.0")
#else
print("no go to 6.0")
#endif
> ./swift-DEVELOPMENT-SNAPSHOT-2024-07-29-a-ubi9/usr/bin/swiftc check.swift  && ./check
no go to 6.0
> ./swift-6.0-DEVELOPMENT-SNAPSHOT-2024-08-02-a-ubi9/usr/bin/swiftc check.swift  && ./check
no go to 6.0
```
Of course, this can be worked around by explicitly setting the language version:
```
> ./swift-DEVELOPMENT-SNAPSHOT-2024-07-29-a-ubi9/usr/bin/swiftc check.swift -swift-version 6 && ./check
got to 6.0
```

Reply:
This is expected, the Swift 6.0 language mode is not enabled by default. Developers are required to enable Swift 6 language mode by using `-swift-version 6.0` 

https://www.swift.org/migration
> Important
> The Swift 6 language mode is opt-in. Existing projects will not switch to this mode without configuration changes.

Reply:
OK, I now see [this was discussed on the forum months ago](https://forums.swift.org/t/default-swift-language-version-for-a-given-toolchain-version/71100), which I missed: I will ask about it there.